### PR TITLE
adding the option of drawing a background to the edges.

### PR DESCRIPTION
**Pull request by yoavdamri**
**Thank you for contributing to vis.js!!**

Please make sure to check the following requirements before creating a pull request:

* [V] All pull requests must be to the [develop branch](https://github.com/almende/vis/tree/develop). Pull requests to the `master` branch will be closed!
* [V] Make sure your changes are based on the latest version of the [develop branch](https://github.com/almende/vis/tree/develop). (Use e.g. `git fetch && git rebase origin develop` to update your feature branch).
* [V] Provide an additional or update an example to demonstrate your changes or new features.
* [V] Update the documentation if you introduced new behavior or changed existing behavior.

### CODE REVIEW

**Comment by wimrijnders** [comment]
Please replace the initial comment with your won description, so that we know what it's about.

File: lib/network/modules/components/edges/util/EdgeBase.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 let util = require("../../../../../util");
 let EndPoints = require("./EndPoints").default;
 
-
```
Comment (mojoaxel) [inline lib/network/modules/components/edges/util/EdgeBase.js]: @yoavdamri Could you please only change the lines of code that really needs to be changed. Syntax "improvements" should not be mixed with other pull requests. The only change in this file should be the `drawBackground` and the `setStrokeDashed` functions.
I know this was probably your editor that did those changes but maybe you can teach it not to this!? Thanks

File: examples/network/edgeStyles/background.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!doctype html>
```
Comment (mojoaxel) [inline examples/network/edgeStyles/background.html]: @yoavdamri Thanks for creating an example! 🥇

File: docs/network/edges.html [hunk 3]
```diff
@@ -4,66 +4,115 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8">
```
Comment (mojoaxel) [inline docs/network/edges.html]: Puh! You really should not change every line in this document! Please only add your lines. It's impossible to review like this.

**Comment by yoavdamri** [comment]
Sorry for the unwanted changes i commit, it is because of the idea auto formatting.
i did the changes, anything else?

**Comment by mojoaxel** [comment]
LGTM!
@yoavdamri Nice Job! Well done. Thanks!

**Comment by wimrijnders** [comment]
But still, please change the initiali comment, so we know what it's about!

**Comment by yoavdamri** [comment]
@wimrijnders i don't think i have to option to do it. is there a document describes howto do it?

**Comment by wimrijnders** [comment]
You sure about that? You are the owner of the comment, you should be able to click on the pencil icon to change it.

I you really can't change it, make a new comment with a description of the PR and I'll paste it in.

**Comment by wimrijnders** [comment]
@yoavdamri I have to say you did a wonderful job. Thanks for updating the documentation as well and adding a demo. Not adding a unit test can be forgiven, because your addition is a visual thing.

Please do come back for more changes!

**Pull request merged**